specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -502,6 +502,64 @@
         }
       }
     },
+    "AcsEmailDeliveryReportReceivedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.EmailDeliveryReportReceived event.",
+      "type": "object",
+      "properties": {
+        "sender": {
+          "description": "The Sender Email Address",
+          "type": "string"
+        },
+        "recipient": {
+          "description": "The recipient Email Address",
+          "type": "string"
+        },
+        "messageId": {
+          "description": "The Id of the email been sent",
+          "type": "string"
+        },
+        "status": {
```

Comment:
Is there a set of known statuses that we expect or is this free form?

Reply:
Not sure is its a set of known or free form. Event service accepts a string for the status. but I will investigate

Reply:
I added the set of know status as enum